sculture-rest/src/main/java/sculture/controllers/SCultureRest.java

```diff
@@ -279,8 +278,7 @@ public BaseStoryResponse story_create(@RequestBody StoryCreateRequestBody reques
             tagStoryDao.update(tagStory);
         }
 
-        List<String> tag_titles = tagStoryDao.getTagTitlesByStoryId(story.getStory_id());
-        return new BaseStoryResponse(story, tag_titles, current_user.getUsername(), current_user.getUsername());
+        return new BaseStoryResponse(story, tagStoryDao, userDao);
```

Comment:
Setting usernames and tags are moved to the Response class. The dao classes is provided for retrieval. I have tried to access dao classes from response model without giving as a parameter but I failed.